collect_app/src/main/java/org/odk/collect/android/audio/VolumeBar.java

```diff
@@ -0,0 +1,121 @@
+package org.odk.collect.android.audio;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.jetbrains.annotations.NotNull;
+import org.odk.collect.android.R;
+
+import static androidx.core.content.res.ResourcesCompat.getDrawable;
+import static org.odk.collect.android.utilities.ViewUtils.dpFromPx;
+import static org.odk.collect.android.utilities.ViewUtils.pxFromDp;
+
+public class VolumeBar extends LinearLayout {
+
+    public static final int MAX_AMPLITUDE = 22760;
```

Comment:
Could you please comment on this to make it clear where the value comes from? My understanding is that amplitude is a positive short so its range is between 0 and [Short.MAX_VALUE](https://docs.oracle.com/javase/7/docs/api/java/lang/Short.html#MAX_VALUE) (32767). If you did mean to use the highest possible value, I think I'd still create the constant for semantics but set it to Short.MAX_VALUE so it's really clear where it comes from.

Reply:
Good catch. I'm using the max value that `AudioRecordView` rather than the max value I've realized. Definitely better to just use `Short.MAX_VALUE` here.